rest_framework/request.py

```diff
@@ -380,11 +402,6 @@ def __getattribute__(self, attr):
         try:
             return super(Request, self).__getattribute__(attr)
         except AttributeError:
-            # Do not proxy attribute access for `user` and `auth`, as this
-            # potentially hides AttributeError's raised in `_authenticate`.
-            if attr in ['user', 'auth']:
-                raise
-
```

Comment:
+1 For not doing this. 🙂

Reply:
Yep, wanted to demonstrate that this does not work.